src/service/command/meme/takopi.ts

```diff
@@ -1,17 +1,30 @@
 import type { MemeTemplate } from '../../../model/meme-template.js';
 
 const takopiFlags = ['f'] as const;
+const takopiOptions = ['c'] as const;
 
-export const takopi: MemeTemplate<typeof takopiFlags[number], never> = {
+export const takopi: MemeTemplate<
+  typeof takopiFlags[number],
+  typeof takopiOptions[number]
+> = {
   commandNames: ['takopi'],
   description:
-    '「〜、出して」\n`-f` で教員と自分の名前の位置を反対にします。([idea: フライさん](https://github.com/approvers/OreOreBot2/issues/90))',
+    '「〜、出して」\n`-f` で教員と自分の名前の位置を反対にします。\n`-c`で教員の名前も変更可能です。\n([idea: フライさん](https://github.com/approvers/OreOreBot2/issues/90))',
```

Comment:
オプション使用時の引数の挙動がわかりにくいので文言、追加しましょう

```suggestion
    '「〜、出して」\n`-f` で教員と自分の名前の位置を反対にします。\n`-c <教員の名前> <出すもの>`で教員の名前も変更可能です。\n([idea: フライさん](https://github.com/approvers/OreOreBot2/issues/90))',
```